config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+  get 'static_pages/index'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  root "static_pages#index"
+
```

Comment:
- [ ] [REQUIRED] Good job so far! A controller with one action should be created and a view for the `root` **route** is created like this  => `root#index`. It should include only `<div id="root">` The rest should be done with React and Redux. Please implement my suggestions 🙏🏼

Reply:
Thanks @mukhammadrizooff, Just fixed